doc/articles/updating-to-winui3.md

```diff
@@ -1,37 +1,41 @@
 # Updating an Uno application to WinUI 3.0
 
-The support for WinUI 3.0 is currently in preview for the Uno Platform. It is currently generated from the WinUI 2.0 / UWP codebase, by applying a set of transformations to align with WinUI 3.0 APIs using the `Uno.WinUIRevert` tool.
+<!-- The support for WinUI 3.0 is currently in preview for the Uno Platform. It is currently generated from the WinUI 2.0 / UWP codebase, by applying a set of transformations to align with WinUI 3.0 APIs using the `Uno.WinUIRevert` tool. -->
+
+Uno Platform supports authoring apps using [WinUI 3's API](uwp-vs-winui3.md). This article details the changes required when migrating an application from the UWP API set to the WinUI 3 API set.
 
 [Read more about WinUI 3 and Uno Platform.](uwp-vs-winui3.md)
 
 ## Migrating an app to WinUI 3.0
 
-- Nuget updates
+- NuGet updates
     - `Uno.UI` becomes `Uno.WinUI`
     - `Uno.UI.RemoteControl` becomes `Uno.WinUI.RemoteControl`
     - `Uno.UI.Lottie` becomes `Uno.WinUI.Lottie`
     - `Uno.UI.DualScreen` becomes `Uno.WinUI.DualScreen`
 - String replacements:
     - `Windows.UI.Xaml` becomes `Microsoft.UI.Xaml`
     - `Windows.UI.Composition` becomes `Microsoft.UI.Composition`
-- Loggers updates
-```
-{ "Windows", LogLevel.Warning },
-{ "Microsoft", LogLevel.Warning },
-```
+- [Logger filter](logging.md) updates
+    ```
+    { "Windows", LogLevel.Warning },
+    { "Microsoft", LogLevel.Warning },
+    ```
```

Comment:
This has changed in the recent templates :
https://github.com/unoplatform/uno/blob/d51c4cdae173c35e4b29c20b54d11980f1b8e23c/src/SolutionTemplate/UnoSolutionTemplate/Shared/App.xaml.cs#L144-L145

and is not needed.

Reply:
App.xaml.cs has to be updated as well mainly a Window object has to be created and the Suspending event has to be adjusted.

See : https://github.com/unoplatform/uno/blob/master/src/SolutionTemplate/UnoSolutionTemplate/Shared/App.xaml.cs#L37
and
https://github.com/unoplatform/uno/blob/master/src/SolutionTemplate/UnoSolutionTemplate/Shared/App.xaml.cs#L56

`Colors` and `ColorHelper` now reside in `Microsoft.UI`. `Color` stays in `Windows.UI`

There is also friction with `Windows.UI.Text` and `Microsoft.UI.Text`.